skflow/__init__.py

```diff
@@ -51,11 +51,12 @@ class TensorFlowEstimator(BaseEstimator):
 
     def __init__(self, model_fn, n_classes, tf_master="", batch_size=32, steps=50, optimizer="SGD",
                  learning_rate=0.1, tf_random_seed=42, continue_training=False,
-                 log_device_placement=True):
+                 log_device_placement=True, verbose=1):
```

Comment:
@terrytangyuan Thank you for being so clear, I'm still a newbie and it is my first contribution :D. So I just rebased the upstream Master, is already everything ok?